modules/apps/object/object-test/src/testFunctional/tests/ObjectModelBuilder.testcase

```diff
@@ -21,11 +22,75 @@ definition {
 		}
 	}
 
+	@description = "LPS-148856 - Verify an error message is seen when attempting to publish invalid object definitions in model builder view"
+	@priority = 5
+	test CannotPublishObjectDefinitionsWithErrors {
+		task ("Given a custom object definition with a decimal field, Object A") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 198013 A",
+				objectName = "CustomObject198013A",
+				pluralLabelName = "Custom Objects 198013 A");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Decimal",
+				fieldLabelName = "Custom Decimal Field",
+				fieldName = "customDecimalField",
+				fieldType = "Double",
+				isRequired = "false",
+				objectName = "CustomObject198013A");
+		}
+
+		task ("and Given two custom object definitions without any custom fields, Object B and Object C") {
+			for (var letter : list "B,C") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = "Custom Object 198013 ${letter}",
+					objectName = "CustomObject198013${letter}",
+					pluralLabelName = "Custom Objects 198013 ${letter}");
+			}
+		}
+
+		task ("and Given the user is in the model builder view") {
+			ObjectModelBuilder.openObjectFolderInModelBuilder();
+		}
+
+		task ("When the user publishes all of the custom object definitions") {
+			Button.clickPublish();
+
+			for (var objectDefinitionName : list "Custom Object 198013 A,Custom Object 198013 B,Custom Object 198013 C") {
+				Check.checkNotVisible(
+					locator1 = "ObjectModelBuilder#CONFIRM_PUBLISHING_OBJECT_DEFINITION_INPUT",
+					objectDefinition = ${objectDefinitionName});
+			}
```

Comment:
@davisaints have you tried running this test without having to do this assertion?

Reply:
It's not a assertion, here I am selecting objects that I want to publish. I'm using the same steps as publishObjectDefinitions macro, but I can't use it since I need to assert the error message before the macro close the modal.